grafana/containers/pgbackrest.json

```diff
@@ -104,28 +188,41 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "ccp_backrest_last_incr_backup_time_since_completion_seconds{pg_cluster=\"[[cluster]]\", role=\"master\"}",
+          "expr": "min(ccp_backrest_last_incr_backup_time_since_completion_seconds{pg_cluster=\"[[cluster]]\", role=\"master\"}) without(deployment,instance,ip,pod)",
           "format": "time_series",
+          "instant": false,
+          "interval": "",
           "intervalFactor": 1,
-          "legendFormat": "Full",
+          "legendFormat": "Incremental Backup",
           "refId": "A"
         },
         {
-          "expr": "ccp_backrest_last_diff_backup_time_since_completion_seconds{pg_cluster=\"[[cluster]]\", role=\"master\"}",
-          "legendFormat": "Diff",
+          "expr": "min(ccp_backrest_last_diff_backup_time_since_completion_seconds{pg_cluster=\"[[cluster]]\", role=\"master\"}) without(deployment, instance,ip,pod)",
+          "hide": false,
+          "interval": "",
+          "legendFormat": "Differential Backup",
           "refId": "B"
         },
         {
-          "expr": "ccp_backrest_last_full_backup_time_since_completion_seconds{pg_cluster=\"[[cluster]]\", role=\"master\"}",
-          "legendFormat": "Incr",
+          "expr": "min(ccp_backrest_last_full_backup_time_since_completion_seconds{pg_cluster=\"[[cluster]]\", role=\"master\"}) without(deployment, instance,ip,pod)",
+          "hide": false,
+          "interval": "",
+          "legendFormat": "Full Backup",
           "refId": "C"
+        },
+        {
+          "expr": "min(pg_stat_archiver_last_archive_age{pg_cluster=\"[[cluster]]\", role=\"master\"}) without(deployment, instance,ip,pod)",
```

Comment:
I'm curious how you are getting this statistic from the container? in pgMonitor we disable all of the metrics that come with postgres_exporter because there were many issues with them not keeping up with changes and they also duplicated many metrics we're already collecting in different, more efficient means.

We don't have a count metric for the WAL archives. We have a size metric with `ccp_wal_activity_total_size_bytes`. The only metric we have for failed WAL archives is `ccp_archive_command_status_seconds_since_last_fail` which is just a time since last failure, so we don't have a count on failure.

Reply:
Actually, for the backrest dashboard we should get getting the archive info from backrest itself. I'll see if I can get that pulled out.

Reply:
So backrest itself does not store the last archive time and it would be difficult to get count metrics out of it since all it presents in the `info` is the first and last wal file.

I've added the metrics you're using here to our `ccp_archive_command_status` metric in my PR

https://github.com/CrunchyData/pgmonitor/pull/233/commits/c825d5dafa0e60290761ad3fe4145a6358828ded

Reply:
Also we don't have this metric on the backrest dashboard because it cannot actually be collected from backrest nor is it unique to being a backrest metric. This is a general postgres metric so we have it on the PG Details dashboard

Reply:
default metrics are not disabled for containers. 
I switched to using ccp_* metrics